api/v1alpha3/openstackcluster_types.go

```diff
@@ -30,7 +30,6 @@ const (
 
 // OpenStackClusterSpec defines the desired state of OpenStackCluster.
 type OpenStackClusterSpec struct {
-
```

Comment:
unrelated change..

Reply:
These are changes that `golangci-lint` requested after I upgrade it. In this case the structs were not formatted with gofumpt